test/spoof/error_checks.go

```diff
@@ -30,14 +29,13 @@ func isTCPTimeout(e error) bool {
 }
 
 func isDNSError(err error) bool {
-	if err, ok := err.(*url.Error); err != nil && ok {
-		if err, ok := err.Err.(*net.OpError); err != nil && ok {
-			if err, ok := err.Err.(*net.DNSError); err != nil && ok {
-				return true
-			}
-		}
+	if err == nil {
+		return false
 	}
-	return false
+	// Checking by casting to url.Error and casting the nested error
+	// seems to be not as robust as string check.
+	msg := strings.ToLower(err.Error())
+	return strings.Contains(msg, "no such host") || strings.Contains(msg, ":53")
```

Comment:
can we add a sample err string in the comment? I would like to understand what :53 means here

Reply:
Done.  :53 is the default port for DNS

Reply:
thanks :+1: